react-common/components/controls/Textarea.tsx

```diff
@@ -57,6 +61,10 @@ export const Textarea = (props: TextareaProps) => {
         if (onChange) {
             onChange(newValue);
         }
+        if (autoResize && resizeRef.current) {
```

Comment:
What happens if someone says autoResize = true but doesn't pass in a resizeRef?

Reply:
Yeah,  I was  thinking about this, too. It felt weird to me, though, that just passing a `resizeRef` would just immediately make the textarea resizable. As I'm typing this, though, I think I was just being a bit overcautious. I think just having the `resizeRef` is fine, and that can be the source of truth for knowing that a we want the textarea to auto-resize.

Reply:
Oh, I hadn't tested that, but just did. If you don't pass in a ref, you get a bunch of errors. I think I see what you're saying, though. We shouldn't need to pass in a ref, and should only need the flag because the ref is just the HTML textarea. Let me play with it.

Reply:
Got rid of it as a prop, now just defining the ref in the component.